extension/observer/ecsobserver/internal/errctx/value.go

```diff
@@ -126,9 +126,11 @@ type valuesError struct {
 
 func (e *valuesError) Error() string {
 	// NOTE: in order to have a consistent output, we sort the keys
-	var keys []string
+	keys := make([]string, len(e.values))
```

Comment:
You can do `make([]string, 0, len(e.values)` and then you can use `append` without needing to managing the increment value.

Reply:
thanks, done